### PR TITLE
Security channel-based detections

### PR DESCRIPTION
**Pull request by JakePeralta7**
Just a proof of concept for detecting some anomalies in event logs even if Sysmon is not deployed across the organization - but the audit policy is configured correctly.

### CODE REVIEW

File: rules/indicator_removal/T1070.009 - Scheduled Task was Deleted.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: T1070.009 - Scheduled Task was Deleted
+group: Indicator Removal
+description: A scheduled task was deleted
+authors:
+  - JakePeralta
+
+
+kind: evtx
+level: critical
```
Comment (FranticTyping) [inline rules/indicator_removal/T1070.009 - Scheduled Task was Deleted.yml:10]: @JakePeralta7 - can you drop this to INFO please. I don't consider a scheduled task being deleted as a clear indicator of malicious activity :)

File: rules/lateral_movement/T1021.004 - Lateral Movement via SSH.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+title: T1021.004 - Lateral Movement via SSH
+group: Lateral Movement
+description: SSH traffic was detected
+authors:
+  - JakePeralta
+# Requires 'Audit Filtering Platform Connection' to be turned on in the GPO
+
+
+kind: evtx
+level: critical
```
Comment (FranticTyping) [inline rules/lateral_movement/T1021.004 - Lateral Movement via SSH.yml:11]: Same as above, INFO please.

File: rules/persistence/T1053.005 - Scheduled Task was Created.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+---
+title: T1053.005 - Scheduled Task was Created
+group: Persistence
+description: A new scheduled task was created
+authors:
+  - JakePeralta
+
+
+kind: evtx
+level: critical
```
Comment (FranticTyping) [inline rules/persistence/T1053.005 - Scheduled Task was Created.yml:10]: Same as above, change to INFO please

File: rules/persistence/T1547.004 - Winlogon System Shell Changed.yml [hunk 4]
```diff
@@ -0,0 +1,48 @@
+---
+title: T1547.004 - Winlogon System Shell Changed
+group: Persistence
+description: The process userinit.exe launched a different process than explorer.exe which indicates the Winlogon\Shell registry value was changed.
+authors:
+  - JakePeralta
+# Requires 'Audit process tracking' to be turned on in the GPO
+
+kind: evtx
+level: critical
```
Comment (FranticTyping) [inline rules/persistence/T1547.004 - Winlogon System Shell Changed.yml:10]: I like tis one, let's change this to MEDIUM level please

**Review by FranticTyping — COMMENTED** [review]
@JakePeralta7 - Thanks for raising this PR! I've left a couple of comments and then I'll be happy to merge

**Comment by JakePeralta7** [comment]
LOL, didn't pay attention to the level attribute. fixed it

**Pull request merged**